WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -279,7 +293,7 @@ private extension ZendeskUtils {
             let logFileInformation = fileLogger.logFileManager.sortedLogFileInfos.first,
             let logData = try? Data(contentsOf: URL(fileURLWithPath: logFileInformation.filePath)),
             let logText = String.init(data: logData, encoding: .utf8) else {
```

Comment:
Thanks @jleandroperez . I have two outstanding PRs branched off of this one - I'll fix those last two items in one of those since they're so minor. I'm merging this thing! 😄